### PR TITLE
[ missioncontrol ] - Support masterKey and joinKey as secrets

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Support masterKey and joinKey as secrets

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/mission-control/values.yaml [hunk 1]
```diff
@@ -306,11 +306,17 @@ missionControl:
...
   ## You can generate one with the command:
-  ## 'openssl rand -hex 16'
-  ## Pass it to helm with '--set missionControl.mcKey=${MC_KEY}'
-  ## IMPORTANT: You should NOT use the example mcKey for a production deployment!
-  mcKey: bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+  ## 'openssl rand -hex 32'
+  ## Pass it to helm with '--set missionControl.masterKey=${MASTER_KEY}'
+  ## IMPORTANT: You should NOT use the example masterKey for a production deployment!
+  masterKey: bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+  mcKey: "{{masterKey}}"
```
Comment (rimusz) [inline stable/mission-control/values.yaml:313]: mcKey: "{{masterKey}}" not going to work

File: stable/mission-control/README.md [hunk 2]
```diff
@@ -147,7 +163,7 @@ In cases where a new version is not compatible with existing deployed version (l
   ```
 * A new mc.key will be generated after this upgrade, save a copy of this key. **NOTE**: This should be passed on all future calls to `helm install` and `helm upgrade`!
 ```bash
-export MC_KEY=$(kubectl exec -it <mission-control-pod> -n <new_namespace> -c mission-control -- cat /var/opt/jfrog/mission-control/data/security/mc.key)
+export MC_KEY=$(kubectl exec -it <mission-control-pod> -n <new_namespace> -c mission-control -- cat /var/opt/jfrog/mission-control/data/security/mc.key )
```
Comment (amithins) [inline stable/mission-control/README.md:166]: Missed changing mc.key to master.key.

Comment (amithins) [reply stable/mission-control/README.md:166]: or is this old version upgrade doc?

Comment (chukka) [reply stable/mission-control/README.md:166]: @amithins  its old version upgrade doc

File: stable/mission-control/values.yaml (hunk 1 above)
Comment (chukka) [reply stable/mission-control/values.yaml]: Removed mcKey from values.yaml

**Pull request merged**